### DISCUSSION TITLE
Disable zram / enable swap

### DISCUSSION THREAD

**Discussion by stefan-sch** [SpiralLinux/SpiralLinux-project#345] [Q&A] [answered]
On installation of SpiralLinux_Plasma_12.231008_x86-64.iso I had to choose "Replace a partiton".
So I couldn't opt for "Swap (with Hibernate)" (which is only available when you erase the complete disk).
Is there a way to disable zram (permanently) and enable swap to a partition (on boot up) to get hibernation, after installation?

**Comment by geckolinux** [comment, marked as answer]
>  Is there a way to disable zram (permanently) and enable swap to a partition (on boot up) to get hibernation, after installation?

Hi there, yes that is possible, but it might be easier to just re-install it with the manual partitioning option, and simply create a swap partition that is slightly bigger than your RAM size, and then it will work for hibernation.

To disable zram: `sudo systemctl disable zramswap.service`

**Reply by stefan-sch** [reply]
Thank you!
